### ISSUE TITLE
[Feature request] Support nx-ovlloader

### ISSUE THREAD

**Issue by benwa** [exelix11/SysDVR#86]
**Is your feature request related to a problem? Please describe.**
To quickly enable and disable the sysmodule, support [WerWolv/nx-ovlloader](/WerWolv/nx-ovlloader)

**Describe the solution you'd like**
Adding `atmosphere/contents/00FF0000A53BB665/toolbox.json` with the following enables support
```json
{
	"name"  : "SysDVR",
	"tid"   : "00FF0000A53BB665",
	"requires_reboot": true
}
```
switch `requires_reboot` to `false` if it doesn't actually need that.

**Comment by exelix11** [comment]
Is adding a toolbox.json file all that's needed ? If so i will do it.

It's not possible to disable SysDVR without a reboot: according to the [documentation on switchbrew](https://switchbrew.org/wiki/GRC_services#Initialize) initializing twice the video capture feature of the OS will hang, even from different sessions.
This means once you disable the sysmodule you can't enable it again or it will crash.
This is the reason why i provide the SysDVR-settings app, it allows to disable streaming to free network/usb/cpu resources while allowing to resume it at any time, only downside is that you can't free the memory it uses so i've been striving to reduce memory usage.

**Comment by benwa** [comment]
Yup, just needs `toolbox.json` to work.

**Comment by exelix11** [comment]
👍 It's coming for the next release then.

**Comment by exelix11** [comment]
The just released 5.0 update includes this.
